communications.md

```diff
@@ -2,44 +2,79 @@
 
-Our communication tools encompasses both formal and informal communications so we may want to consider the following factors:
-* Your audience (Is this important to the Locus team or Atom team?)
-* The size of the audience (Is this relevant to one person or a whole team?)
-* Content of message and reusability (Will this message be useful to someone else in the future?)
+Our communication tools encompass both formal and informal communications so we may want to consider the following factors:
+* Your audience
+	* Who and When - be sensitive to the workflows of others
+	* Size - is this relevant to one person or a whole team?
+	* What - Locus, Atom, Data or Product
+* Content of message and reusability
+	* Will this message be useful to someone else in the future?
+
+In general, if it’s applicable to more than one person, share in Slack, Asana, Drive or Miro so it can be searchable and found in an accessible manner. For example, when getting into a longer discussion in Slack, respond in a Thread and change the first message to include, "*Thread: [topic of the thread]*", for easy searching and, in the future, parsing. Within the thread, add tags to messages, such as "*Summary: ...*", "*Question: ...*", and "*TODO: ...*", when useful.
 
-In general, if it’s applicable to more than one person, share in the relevant Slack channel or Asana project so it can be searchable and found in an accessible manner. For example, asking questions in a Slack channel allow responses to those questions to become part of our knowledge base. Only exception is sharing common credentials or secrets - we have a well-controlled private channel for this.
+Each person communicates and receives information differently, so you are encouraged to find out (Slack status or ask) how someone would be like to be contacted. e.g. ping on Slack vs. Asking in person.
 
-Each person communicates and receives information differently so if something requires attention, feel free to verbally ask colleagues or check their Slack status for how they would be like to be communicated. e.g. ping on Slack versus asking in person.
+When in doubt, use Slack.
 
-When there is doubt, use Slack.
+## Practices
+Intertwined amongst all of these tools are some common usages that have developed over time. They are not rules, rather practices some have agreed make their lives easier and as such are mostly **opt-in**. Some examples are:
+
+ - Dev Journal
+ - Slack Threads
+ - Asana + Google Drive Knowledge Base
+
+All of these are open to change since their goal is purely to optimize our work environment. If you have suggestions for how to improve them or have your own you'd like to try out, go for it. While doing so, remember to be the champion of your own cause. Since you're the person trying to invoke change, lead the way. Suggest it to the team, show them how it adds value and encourage them to try it out. Do not just throw out an idea and expect it to catch fire. Be persistent.
```

Comment:
The intention I get from this is to empower people to share & try out ideas to optimize our work environment. But it may come off as lecturing with the line "Do not just throw out an idea ...".

Suggestion: "If you have suggestions for how to improve these practices or have your own idea, feel free to try it out, share it with the team and show how it can add value."